pkg/utils/printableresource.go

```diff
@@ -117,45 +130,243 @@ func (res *PrintableResource) DumpResource() [][]string {
 	}
 
 	data := [][]string{}
-	var row []string
 	switch res.verboseType {
 	case "keyinfo":
-		keyInfo := res.dumpSubTable(res.keyInfo, false)
-		if len(keyInfo) == 0 {
-			row = []string{paddingFirstLine + res.typeName, res.name, ""}
+		data = res.dumpToMultipleRows(res.keyInfo, paddingFirstLine, paddingSubLines, []string{res.typeName, res.name}...)
+	default:
+		data = res.dumpToMultipleRows(res.conditions, paddingFirstLine, paddingSubLines, []string{res.typeName, res.name, res.createdAt}...)
+	}
+
+	return data
+}
+
+func (res *PrintableResource) dumpToMultipleRows(elements [][]string, paddingFirstLine, paddingSubLines string, fixedColumns ...string) [][]string {
+
+	data := [][]string{}
+	var row []string
+	subtableRows := res.dumpSubTable(elements, false)
+	//empty conditions, we need to copy the fixed columns
+	if len(subtableRows) == 0 {
```

Comment:
Break this complex if/else into different smaller functions. For instance extract the if statements into their one function and same for the else. So this will be simple if/else and calling the function. This simplifies logic a lot.

Of course, feel free to simplify differently but in general I would avoid nested if/else and long if/else